Add more detailed steps for beginner (#39)
ignitionrobotics/docs: contributing.md

1. **Fork the Ignition library** you want to contribute to. This will create
   your own personal copy of the library. All of your development should
   take place in your fork.

>    - An important thing to do is create an remote pointing to the [upstream remote repository](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork). This way, you can always check some modification on the original repository and **always** keep your fork repository updated.

Reviewer: ```suggestion
   - An important thing to do is create a remote pointing to the [upstream remote repository](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork). This way, you can always check for modifications on the original repository and **always** keep your fork repository up to date.
```

   a release / master branch. This is a good habit to get in, and will make
   your life easier.

> 1. **Write your code.** This is the fun part, but is good to remeber:
>    - Always [sign your commits](https://docs.github.com/en/github/authenticating-to-github/signing-commits);
>    - Look the existing code and try to keep the pattern used as much as possible;
>    - **Alway** keep your branch updated with the original repository;

Reviewer: ```suggestion
1. **Write your code.** This is the fun part, but is good to remember:
   - Always [sign your commits](https://docs.github.com/en/github/authenticating-to-github/signing-commits) (See the bullet about Developer Certificate of Origin in the [Process](#markdown-header-process) section below)
   - Look at the existing code and try to maintain the existing style and pattern as much as possible
   - **Always** keep your branch updated with the original repository
```
